modules/jtframe/devops/docker-installer.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# This script will remove your existing Docker installation and
+# will do a fresh Docker installation, including official plugins
+# such as composer and buildx (required for building Jotego images).
+# It will also set Docker in rootless mode so you won't need to use
+# sudo, either interactively or within scripts.
+# The script is architecture-independent.
+
+set -e
+
+echo "🚀 Docker installation on Ubuntu for Jotego devops..."
+
+DOCKER_GPG_DIR="/etc/apt/keyrings"
+DOCKER_GPG="$DOCKER_GPG_DIR/docker.gpg"
+ARCH=$(dpkg --print-architecture)
+RELEASE=$(lsb_release -cs)
+
+# 1. Remove any existing Docker installation
+echo "🔄 Removing old Docker versions..."
+sudo apt-get remove -y docker docker-engine docker.io containerd runc || true
+sudo apt-get purge -y docker docker-engine docker.io containerd runc || true
+sudo rm -rf /var/lib/docker /var/lib/containerd || true
+
+# 2. Install dependencies
+echo "📦 Installing dependencies..."
+sudo apt-get update
+sudo apt-get install -y ca-certificates curl gnupg lsb-release uidmap dbus-user-session
+
+# 3. Add Docker GPG key and official repository
+echo "🔑 Adding Docker GPG key..."
```

Comment:
Do not have `...` at the end of each line. It clutters the screen with no real information. Maybe leave it for cases where you know that it is going to take a long time (several minutes) to run the command.